api/src/main/java/org/apache/iceberg/Table.java

```diff
@@ -65,6 +65,20 @@
    */
   Map<Integer, PartitionSpec> specs();
 
+  /**
+   * Return the {@link SortOrder sort order} for this table.
+   *
+   * @return this table's sort order
+   */
+  SortOrder sortOrder();
+
+  /**
+   * Return a map of {@link SortOrder sort orders} for this table.
```

Comment:
I think when we return a Map of common classes we should describe what they represent, the Integer keys here I assume are Field numbers right?

Reply:
It is a map from order id -> order. Let me add a comment.

Reply:
Done.